grails-app/services/au/org/ala/merit/ProjectService.groovy

```diff
@@ -1125,10 +1128,19 @@ class ProjectService  {
 
     /**
      * Returns true if project reports are allowed to be regenerated.
+     * This allows project dates to be changed while a project is in the Application status
      *  @param project the project to check, expects the reports property to have
      * been populated with project reports.
      */
     boolean canRegenerateReports(Map project) {
```

Comment:
Aside from being confusing (this method returns true when you should not be able to regenerate reports - i.e. the project is in application status) this is also wrong.  If you change a reporting period in the MU settings, projects in application status will have reports generated for them which is incorrect behaviour.